debugger.c

```diff
@@ -12,7 +12,7 @@
 #include <stdlib.h>
 #include <stdio.h>
 #include <inttypes.h>
-#include <SDL2/SDL.h>
+#include "SDL.h"
```

Comment:
```suggestion
#include <SDL.h>
```
(and elsewhere)

Reply:
It isn't a "system header"; it's a "user header".  That usage isn't about where the file sits in the file-system.  It's about how the preprocessor knows where to find it.  System directories already are known to the preprocessor.  But, "/usr/include/SDL2/" isn't known; the command line uses a `-I` option to tell the preprocessor where to look.  That makes it "a user-specified path".

Reply:
That distinction is new to me.
By my knowledge, user headers are files of the current project, other wise system.
I thought that's why we have `-I` and `-isystem`.
Can you point me to a text giving reasons for this because I'm not convinced :)

Reply:
I chose quotes as the [SDL FAQ](https://wiki.libsdl.org/FAQDevelopment#Do_I_.23include_.3CSDL.h.3E_or_.3CSDL.2FSDL.h.3E.3F) advises:

> ### Do I #include <SDL.h> or <SDL/SDL.h>?
> 
> The most portable way to include SDL headers is to use quotes around the header name:
> 
> #include "SDL.h"

Reply:
Can't argue against portability :D